Sources/NetworkViewer/Common/Services/ShareService/FileShareProvider.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  FileShareProvider.swift
+//  NetworkViewer
+//
+//  Created by Mакс T on 19.11.2024.
+//
+
+import Foundation
+import UIKit
+
+final class FileShareProvider: ShareProvider {
+    var displayName: String { "file" }
+    var icon: UIImage? = UIImage(systemName: "arrow.down.doc.fill")
+
+    func shareData(for operations: [NetworkViewer.Operation]) -> ShareService.Result? {
+        let combinedContent = operations.map { OperationMapper.jsonFrom($0) }.joined(separator: "\n\n")
```

Comment:
I think we can combine json objects into a new one. So it can looks like:

```
{
  "operations": [
    { ... },
    { ... }
  ]
}
```
So we will be able to paste this text to any json formatter and view formatted json text